Dungeoneer/src/com/interrupt/dungeoneer/gfx/GlRenderer.java

```diff
@@ -1977,7 +1977,7 @@ public void drawTextRightJustified(String text, float sPosX, float sPosY, float
 
 	public void drawCenteredText(String text, float sPosY, float size, Color color, Color shadowColor)
 	{
-		if(text == null) return;
```

Comment:
Are there other places where text is rendered where we should make this same check?

Reply:
Nope, after adding that line the game does not crash by the problem in the topic.